src/adb/forward.js

```diff
@@ -15,5 +14,5 @@ module.exports = function forward(hostPort, remotePort, options = {}) {
   }
 
   debug(cmd);
```

Comment:
Since our `exec` and `spawn` wrappers log the command I think we can drop the command-specific debug statements?